OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -1989,6 +1988,15 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
                 }
             }
 
+            if (first == DreamValue.Null) {
+                // nulls are equal to each other
+                if (second == DreamValue.Null)
+                    return true;
+                // null != 0 or ""
+                if (second.Type is DreamValue.DreamValueType.Float or DreamValue.DreamValueType.String) {
+                    return false;
+                }
+            }
```

Comment:
This is already done in the above switch inside `case DreamValue.DreamValueType.DreamObject:`